third_party/rust/Cargo.toml

```diff
@@ -1308,10 +1308,6 @@ package = "urlencoding"
 path = "utf_8/v0_7/crate"
 package = "utf-8"
 
-[patch.crates-io.uuid_v0_8]
-path = "uuid/v0_8/crate"
-package = "uuid"
-
```

Comment:
If this was the last usage, can also remove the source from `third_party/rust/uuid/v0_8`.